roles/gpg-import/tasks/main.yml

```diff
@@ -29,33 +29,37 @@
     - { name: public.key, src: "{{ gpg_private_key }}" }
     - { name: private.key, src: "{{ gpg_public_key }}" }
     - { name: trust, src: "{{ gpg_trust_file }}" }
-
+  when: gpg_import_keys
 
 - name: Make sure the GPG config exists
   become: yes
   become_user: "{{ gpg_user }}"
   command: "gpg --list-keys"
   args:
     creates: "{{ gpg_home }}/.gnupg"
+  when: gpg_import_keys
```

Comment:
I guess you added this to variable to control when the GPG key is to be imported. Since the backups don't work without the keys, don't you think it would be better to have a variable controlling whether backups are setup or not.

The `when` condition would be placed here:
 - https://github.com/OpenSRP/playbooks/blob/master/roles/couchdb/meta/main.yml#L10
 - https://github.com/OpenSRP/playbooks/blob/master/roles/mysql-5.7/meta/main.yml#L7